tensornetwork/tensor.py

```diff
@@ -111,7 +111,11 @@ def reshape(self, shape: Sequence[int]) -> "Tensor":
     """
     reshaped = self.backend.reshape(self.array, shape)
     return Tensor(reshaped, backend=self.backend)
-
+  def item(self):
+    """Return a Python scalar formed after converting a size-1 Tensor
+        if size of Tensor is not 1
+        then raises ValueError"""
+    return self.array.item()
```

Comment:
I don't think all the backend arrays have this method, so we will have to add an item method to the backends as well. See e.g. `reshape` for how that looks within `Tensor`.